library/src/main/kotlin/com/chuckerteam/chucker/internal/support/ResponseProcessor.kt

```diff
@@ -43,6 +43,7 @@ internal class ResponseProcessor(
             requestDate = response.sentRequestAtMillis
             responseDate = response.receivedResponseAtMillis
             protocol = response.protocol.toString()
+            hostIp = response.body?.source()?.getHostIp()
```

Comment:
I'm having trouble understanding this. Why is it assumed that the response body contains IP?